rest/views.py

```diff
@@ -95,13 +95,7 @@ def get_queryset(self):
 
     def perform_create(self, serializer):
         """Create a new repository."""
-        # pylint: disable=protected-access
-        repo = serializer.save()
-        assign_user_to_repo_group(
-            self.request.user,
-            repo,
-            GroupTypes.REPO_ADMINISTRATOR
-        )
+        serializer.save()
```

Comment:
This is the default behavior of `perform_create` so you should be able to just remove this method